### PR TITLE
Adding new rancher 1.2.14 and 1.3 Portworx(docker container) catalog templates

### PR DESCRIPTION
**Pull request by sensri108**
Adding new rancher 1.2.14 and 1.3 Portworx(docker container) catalog templates. Customer request waiting for this change implementations.

### CODE REVIEW

**Comment by deniseschannon** [comment]
@senpx can you squash your commits? also, if you need this immediately, it might be easier to create a private catalog to provide to your customers int he short term if they need these templates immediately. But by doing a private repo, your customers would not be able to upgrade between the Rancher catalog and the private catalog. 

As this is going in with of our v1.6 development process, it won't be released until we release the next v1.6, which is slated end of february.

File: infra-templates/portworx/0/docker-compose.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ portworx:
     KVDB: ${KVDB}
     HDR_DIR: ${HEADER_DIR}
     USE_DISKS: ${USE_DISKS}
+    ENABLE_ATTR_CACHE: true
```
Comment (deniseschannon) [inline infra-templates/portworx/0/docker-compose.yml:17]: You can't make changes to existing catalog entries as they have already been released to the public.

File: infra-templates/portworx/0/docker-compose.yml [hunk 2]
```diff
@@ -23,4 +24,5 @@ portworx:
      - /opt/pwx/bin:/export_bin
      - /var/run/docker.sock:/var/run/docker.sock
      - /var/cores:/var/cores
+     - /proc:/hostproc
```
Comment (deniseschannon) [inline infra-templates/portworx/0/docker-compose.yml:27]: You can't make changes to existing catalog entries as they have already been released to the public.

**Comment by sensri108** [comment]
Sorry for the confusion. The issue is already resolved. Closing the PR.

**Pull request closed without merging**